### PR TITLE
Add voice urls - registration

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review after landing #5 .

**Comment by imsickofmaps** [comment]
@gsvr lets optimise the Year/Month/Day thing as per Slack chat before re-review

File: src/voice_registration.js [hunk 1]
```diff
@@ -55,154 +66,77 @@ go.app = function() {
...
-            };
+        self.states.add('state_r06_birth_month', function(name) {
+            var speech_option;
+            self.im.user.answers.state_r04_mom_state === 'pregnant'
+                ? speech_option = '1'
+                : speech_option = '2';
+            // create choices eg. new Choice('1', $('1')) etc.
+            var month_choices = [];
+            for (i=1; i<=12; i++) {
+                month_choices.push(new Choice(i.toString(), $(i.toString())));
```
Comment (imsickofmaps) [inline src/voice_registration.js]: I _think_ this will break the translation engine. However, you don't need to translation the digits 1-12 :grinning:

**Comment by imsickofmaps** [comment]
:+1: once the `$(function())` is removed

File: src/utils.js [hunk 2]
```diff
@@ -17,6 +17,92 @@ go.utils = {
...
+    make_speech_url: function(im, name, lang, num) {
+        return im.config.control.url + lang + '/' + name + '_' + num + '.mp3';
+    },
+
+    // Construct helper_data object
+    make_voice_helper_data: function(im, name, lang, num) {
+        return {
+            voice: {
+                speech_url: go.utils.make_speech_url(im, name, lang, num)
+            }
```
Comment (imsickofmaps) [inline src/utils.js:68]: Add `wait_for: '#'` under the url

**Pull request merged**